### PR TITLE
net/wireguard: Allow 100 instances

### LINKED ISSUE #2759: net/wireguard: Local Configuration - Maximum number of instances reached
https://github.com/opnsense/plugins/blob/f8f3975e00560425bd1de3136320d181a83e4f84/net/wireguard/src/opnsense/mvc/app/models/OPNsense/Wireguard/Server.xml#L20

It looks like we are hard-coding the maximum number of WG local configurations to 20.
Can this be increased, or the limitation be removed altogether?
I haven't been able to find any evidence so far on an actual limitation within WG itself (could be lack of googling skill here too, so happy to stand corrected)

### PR DESCRIPTION
**Pull request by mimugmail**
Closes #2759

### CODE REVIEW

**Comment by fichtner** [comment]
Does the upper limit matter in practice? It would be saner to drop it IMO.

**Comment by mimugmail** [comment]
Only a lack of experience, I can also remove it .. reminds me that I forgot to bump the model :)

File: net/wireguard/src/opnsense/mvc/app/models/OPNsense/Wireguard/Server.xml [hunk 1]
```diff
@@ -16,9 +16,6 @@
                     <ValidationMessage>Should be a string between 1 and 32 characters. Allowed characters are 0-9, a-z, and A-Z</ValidationMessage>
                 </name>
                 <instance type="AutoNumberField">
-                    <MinimumValue>0</MinimumValue>
```
Comment (fichtner) [inline net/wireguard/src/opnsense/mvc/app/models/OPNsense/Wireguard/Server.xml:19]: Hopefully -1 isn’t valid input?

Comment (mimugmail) [reply net/wireguard/src/opnsense/mvc/app/models/OPNsense/Wireguard/Server.xml:19]: Its an AutoNumberField without any way to edit via UI, shall Min stay here?

Comment (fichtner) [reply net/wireguard/src/opnsense/mvc/app/models/OPNsense/Wireguard/Server.xml:19]: Maybe @AdSchellevis knows by memory, not at a computer at the moment.

Comment (AdSchellevis) [reply net/wireguard/src/opnsense/mvc/app/models/OPNsense/Wireguard/Server.xml:19]: @mimugmail you should be safe, 0 is the default :)

https://github.com/opnsense/core/blob/af587e812a7b8d96cf3fd84869dc7e00bbfe09b3/src/opnsense/mvc/app/models/OPNsense/Base/FieldTypes/AutoNumberField.php#L71

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**